openssl-sys/src/handwritten/x509.rs

```diff
@@ -645,3 +673,22 @@ extern "C" {
     pub fn X509_print(bio: *mut BIO, x509: *mut X509) -> c_int;
     pub fn X509_REQ_print(bio: *mut BIO, req: *mut X509_REQ) -> c_int;
 }
+
+#[repr(C)]
+pub struct X509_PURPOSE {
+    pub purpose: c_int,
+    pub trust: c_int, // Default trust ID
+    pub flags: c_int,
+    pub check_purpose:
+        Option<unsafe extern "C" fn(*const X509_PURPOSE, *const X509, c_int) -> c_int>,
+    pub name: *mut c_char,
+    pub sname: *mut c_char,
+    pub usr_data: *mut c_void,
+}
+
+const_ptr_api! {
+    extern "C" {
+        pub fn X509_PURPOSE_get_by_sname(sname: #[const_ptr_if(any(ossl110, libressl280))] c_char) -> c_int;
+        pub fn X509_PURPOSE_get0(idx: c_int) -> *mut X509_PURPOSE;
```

Comment:
As this doesn't use the const ptr attribute is should probably go outside the const_ptr_api block for consistency with everything else.

Reply:
Yes. No idea how this happened.